transformations/Makefile

```diff
@@ -0,0 +1,26 @@
+LIBRARY = "library/"
+ALL_SUBFOLDERS = $(LIBRARY) "bq/" "sf/"
```

Comment:
The reference to sf/ directory is generated by setool. Should we remove it from the generator in case just BQ project is selected @Jesus89?

Reply:
We adjust this, but now it's not a problem because there is a check (if [ -d $${s} ];)